tests/test_errors.py

```diff
@@ -812,6 +812,18 @@ def finalize(self, cfg: Any) -> None:
         ),
         id="list:create_not_optional:_set_value(None)",
     ),
+    pytest.param(
+        Expected(
+            create=lambda: ListConfig(content=[1, 2]),
+            op=lambda cfg: cfg._set_value(True),
+            exception_type=ValidationError,
+            object_type=None,
+            msg="Invalid value assigned : True is not a subclass of ListConfig or list",
```

Comment:
Do we always show the object content in those kind of errors?
it could get crowded for large objects. should this show the type instead?

Reply:
ping

Reply:
I don't know about that one. I guess we can show the type.

Reply:
Okay, follow the same pattern here:
```
msg="Invalid type assigned : int is not a subclass of FoobarParams. value: 20",
```

Value last makes it easier to read in case the value is big.